src/modules/app/constants/buildConfig.js

```diff
@@ -2,7 +2,32 @@
 
 import type { BuildConfigType } from '../../../../build-configs/configs/BuildConfigType'
 import loadBuildConfig from '../../../../build-configs'
+import { INTEGREAT_ICONS } from '../../../../build-configs/configs/integreat'
+import { MALTE_ICONS } from '../../../../build-configs/configs/malte'
+import integreatAppLogo from '../../../../build-configs/configs/integreat/assets/app-logo.png'
+import integreatLocationMarker from '../../../../build-configs/configs/integreat/assets/location-marker.svg'
+import malteAppLogo from '../../../../build-configs/configs/malte/assets/app-logo.svg'
+import malteLocationMarker from '../../../../build-configs/configs/malte/assets/location-marker.svg'
 
-const buildConfig = (): BuildConfigType => loadBuildConfig(process.env.BUILD_CONFIG_NAME)
+const buildConfigIcons = (iconSet: string) => {
+  if (iconSet === INTEGREAT_ICONS) {
```

Comment:
You can also check here directly for the appName. There is no need to a separate build config option if we handle it like this imo.

Reply:
integreat-e2e and integreat-test-cms are using the same icons as integreat but have different app names.